HHS/Head-Start-TTADP
# [TTAHUB-981] Fix OE Objectives Saving and Read Only View

## Description of change

This ticket fixes two issues:

1. Error when saving objectives on a other-entity report.
2. Display saved goals in a read only view.

**Update 0907:** To return to the report and see the objectives in read only mode you will need to comment out validation for the fields we currently do NOT save (roles, topics, and resources) - GoalValidator.js -unfinishedObjectives().

## How to test

1. Create an other entity report.
2. Saving objectives on the report should no longer error.
3. Once the objectives are saved you should be redirected to a read only view.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-981


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
